corehq/apps/importer/templates/importer/excel_fields.html

```diff
@@ -107,6 +107,44 @@
             });
 
             $('#autofill').click(autofillProperties);
+
+            function validateSubmitButton() {
+                var invalidFields = $('.excel_field option:selected')
+                                      .filter(function(k, v) { 
+                                          return /\s+/.test(v.value.trim());
+                                      }).length;
+                if (invalidFields > 0) {
+                    $('.form-actions [type=submit]').prop('disabled', true);
+                } else {
+                    $('.form-actions [type=submit]').prop('disabled', false);
+                }
+            }
+
+            function populateRowErrors(tr) {
+                var field = tr.find('.excel_field'), selected;
+
+                if (field.length === 0) { return; }
+
+                selected = field.val().trim();
+                if (/\s+/.test(selected)) {
+                    tr.addClass('error');
+                    tr.find('.errorMsg')
+                      .text('{% trans "This column contains a whitespace character" %}')
```

Comment:
i wonder if this gets picked up as needed to be translated since it's in a JS file